docs/examples/go-experimental/getting-started.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 1
+---
+
+# Getting Started
+
+This page will explain how to structure your genezio project when using GO for the backend language.
+
+### Project Structure
+
+Because go declarations belong to packages instead of files, genezio allows one deployable genezio class per package. This means that for a project with 2 genezio classes, a valid project structure should look like this:
```

Comment:
```suggestion
Because Go declarations belong to packages instead of files, genezio allows one deployable genezio class per package. This means that for a project with 2 genezio classes, a valid project structure should look like this:
```